### PR TITLE
Enhancement: Add support for PHP 7.3

### LINKED ISSUE #459: Add support for PHP 7.3

### PR DESCRIPTION
**Pull request by localheinz**
This pull request

- [x] adds support for PHP 7.3

Fixes #459.

### CODE REVIEW

**Comment by mvorisek** [comment]
Hi @localheinz, do you plan to finish this PR?

**Comment by mvorisek** [comment]
Hi @localheinz, do you plan to finish this PR?

TODO for myself: finish https://github.com/roundcube/roundcubemail/pull/9320 once this PR is merged and released

File: test/Unit/Version/SeriesTest.php [hunk 1]
```diff
@@ -37,7 +37,7 @@ public function testCreateReturnsSeries(): void
     }
 
     /**
-     * @dataProvider \Ergebnis\DataProvider\StringProvider::arbitrary
+     * @dataProvider \\Ergebnis\PHPUnit\SlowTestDetector\Test\DataProvider\StringProvider::arbitrary
```
Comment (mvorisek) [inline test/Unit/Version/SeriesTest.php:40]:
```suggestion
     * @dataProvider \Ergebnis\PHPUnit\SlowTestDetector\Test\DataProvider\StringProvider::arbitrary
```

**Comment by localheinz** [comment]
Released as [`ergebnis/phpunit-slow-test-detector:2.11.0`](https://github.com/ergebnis/phpunit-slow-test-detector/releases/tag/2.11.0), @mvorisek!

**Comment by mvorisek** [comment]
Thank you ❤

**Pull request merged**